### ISSUE TITLE
Allow users to set tile background colors to whatever they want and override both the tile's .desktop file and their accent color for all tiles (if they want, say, black tiles with a light background without using high contrast) and support specifying tile background colors in themes for high contrast theme support.

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/Retiled#233]
I just realized I also need to specify whether a theme is high contrast or not in its file. Guess that's what the theme family file is for.

I got the idea to allow black tiles when reading a WPCentral article a long time ago and it talked about how cool black tiles look but the issue is they can only be used in high contrast mode. I'm also going to need to allow themes to set background colors for tiles, so maybe what I can do is by default only allow high contrast themes to override tile background colors without the user having to toggle something to allow it, and otherwise all themes can override tile background colors if the user allows it, or the user can override it themselves with a color of choice (but that won't be available in high contrast).

**Comment by DrewNaylor** [comment]
I'll need to have another setting for this: whether to override all tile background colors or just the ones that aren't specified in their .desktop files. This setting, along with the setting to allow all themes to override tile background colors (and not just high-contrast ones) will need to be on an advanced theme settings page to not overwhelm anyone. Also maybe the settings to change the color overlay for background wallpapers and its percentage can go there, too. My goal is a balance of the simplicity and ease-of-use from Windows Phone mixed with the advanced customization of KDE. Power users will be able to tweak tons of things and anyone that doesn't want to mess with it too much can just keep with what's presented.

Also themes will need to be able to temporarily override the accent color for high contrast use, but this won't affect the config files and will only be like `if (HighContrastThemeActive) then accentColor = "white" else accentColor = (SavedAccentColor)`.

**Comment by DrewNaylor** [comment]
Tagging with accessibility due to this involving high contrast theme support.
